taskManager/src/main/java/com/kunmi/taskManager/taskManager/exceptions/ExceptionResponseBuilder.java

```diff
@@ -0,0 +1,35 @@
+package com.kunmi.taskManager.taskManager.exceptions;
+
+import java.util.Set;
+
+public class ExceptionResponseBuilder {
```

Comment:
Do your really need this builder? You already have @Builder annotation on ExceptionResponse